package.json

```diff
@@ -27,14 +27,14 @@
   },
   "homepage": "https://github.com/Cimpress-MCP/AT-OpenAPI#readme",
   "dependencies": {
-    "@alphatango/exceptions": "^0.0.4",
     "jsonwebtoken": "8.5.1",
     "openapi-factory": "4.4.226"
   },
   "peerDependencies": {
-    "aws-sdk": "^2.0.0"
+    "@alphatango/exceptions": "^2.0.0"
```

Comment:
Is this equivalent of 2.x.x ?

Reply:
Yes, do we want to limit it to `2.0.x`? Then it would be `~2.0.0` .. btw helpful tool https://semver.npmjs.com/

Reply:
Limited to `2.0.x`